packages/core/src/odata/common/request-builder/batch/batch-request-serializer.ts

```diff
@@ -1,26 +1,33 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
 import voca from 'voca';
+import { ODataRequest } from '../../request/odata-request';
+import { ODataRequestConfig } from '../../request/odata-request-config';
 import { MethodRequestBuilderBase } from '../request-builder-base';
 import { BatchChangeSet } from './batch-change-set';
 import { BatchRequestBuilder } from './batch-request-builder';
-
+import {
+  BatchRequestOptions,
+  BatchRequestSerializationOptions,
+  defaultOptions
+} from './batch-request-options';
 /**
  * Serialize change set to string.
  * @param changeSet - Change set containing a collection of write operations.
- * @param changeSetBoundary Boundary separating parts in the serialized request.
+ * @param options Request serialization options.
  * @returns The serialized string representation of a change set.
  */
 export function serializeChangeSet(
-  changeSet: BatchChangeSet
+  changeSet: BatchChangeSet,
```

Comment:
just to be sure, this method is not part of the public API?

Reply:
Nope